public/locales/auset-te.json

```diff
@@ -0,0 +1,426 @@
+      "setup": "<i>1.</i> నది వెంబడి ఏవైనా క్లియరింగ్‌లలో 4 <strong>యోధులను</strong> ఉంచండి.<br/><i>2.</i> 3 <strong>యోధులను</strong> ఉంచండి. మీ చెల్లింపుల పెట్టె.<br/><i>3.</i> సరిపోలే <strong>ట్రేడ్ పోస్ట్‌లు</strong>తో మీ ట్రేడ్ పోస్ట్‌ల ట్రాక్‌లను పూరించండి.<br/><i>4.</i> మీ 3ని ఉంచండి. మీ సేవల ట్రాక్‌లో <strong>సేవా గుర్తులు</strong>, ప్రతి సేవకు ధరను సెట్ చేస్తుంది."
+    },
+    "vagabond": {
+      "name": "సంచారి(వాగాబాండ్)",
+      "summaryTitle": "వాగాబాండ్ గా ఆడటం",
+      "summary": "<p>వాగాబాండ్‌గా, మీరు సంఘర్షణ యొక్క అన్ని వైపులా ఆడతారు, మీకు సరిపోయే విధంగా స్నేహితులు మరియు శత్రువులను చేసుకుంటారు. మీరు మీ <strong>సంబంధాల</strong> ద్వారా పాయింట్లను స్కోర్ చేస్తారు, మీరు స్నేహపూర్వక వర్గాలకు కార్డ్‌లు ఇవ్వడం ద్వారా సహాయం చేస్తారు మరియు యుద్ధంలో వారి ముక్కలను తీసివేయడం ద్వారా శత్రు వర్గాలతో మీ అపఖ్యాతిని పెంచుకుంటారు. వుడ్‌ల్యాండ్‌లోని జీవులలో మీ మంచి పేరును వ్యాప్తి చేయడానికి మీరు <strong>అన్వేషణలు</strong> చేస్తున్నప్పుడు కూడా మీరు పాయింట్‌లను స్కోర్ చేస్తారు.</p><p>కదలడానికి మరియు సమర్థవంతంగా పని చేయడానికి, మీరు మీని నిర్వహించాలి <strong>వస్తువుల</strong> సాచెల్, పురాతన <strong>శిధిలాలను</strong> అన్వేషించడం ద్వారా మరియు ఇతర వర్గాలకు సహాయం అందించడం ద్వారా మీ ఎంపికను విస్తరించడం. <strong>ఒంటరి వాండరర్</strong>గా, మీరు క్లియరింగ్‌ను పాలించలేరు లేదా మరొక వర్గాన్ని పాలించకుండా ఆపలేరు, కానీ మీరు <strong>చురుకైన</strong>, కాబట్టి మీ క్లియరింగ్‌ను ఎవరు పాలించినా మీరు తరలించవచ్చు.</ p>",
+      "setupTitle": "వాగాబాండ్‌ని ఏర్పాటు చేయండి",
+      "setup": "<i>1.</i> మీ <strong>పాన్</strong>ని ఏదైనా అడవిలో ఉంచండి.<br/><i>2.</i> క్వెస్ట్ డెక్‌ని షఫుల్ చేయండి. సమీపంలో వ్యవహరించడానికి 3 <strong>క్వెస్ట్‌లను</strong> గీయండి.<br/><i>3.</i> <Bag/>, <Boot/>, <Hammer/> మరియు <Sword/> ఉంచండి. <strong>ఐటెమ్‌లను నాశనం చేయండి</strong> <i>(\"R\"తో గుర్తు పెట్టబడింది)</i> శిథిలాల కింద యాదృచ్ఛికంగా, ఇది ఇప్పటికే పూర్తి చేయకపోతే.<br/><i>4.</i> మీ క్యారెక్టర్ కార్డ్ స్లాట్‌లో \"{{vagabond}}\" <strong>అక్షర కార్డ్</strong>ని ఉంచండి. <InitialStartingItems>, </InitialStartingItems> మరియు <FinalStartingItem/> <strong>ప్రారంభ అంశాలను</strong> <i>(\"S\"తో గుర్తు పెట్టబడింది)</i>ని మీ సాట్చెల్‌లో మరియు వాటి మ్యాచింగ్ ట్రాక్‌లలో ఉంచండి, అవసరం మేరకు.",
+      "setup_vagabondSetUp": "<i>1.</i> మీ <strong>పాన్</strong>ని ఏదైనా అడవిలో ఉంచండి.<br/><i>2.</i> క్వెస్ట్ డెక్‌ని షఫుల్ చేయండి. సమీపంలో వ్యవహరించడానికి 3 <strong>క్వెస్ట్‌లను</strong> గీయండి.<br/><i>3.</i> అదనపు <Bag/>, <Boot/>, <Hammer/> మరియు <Sword/ని ఉంచండి > <strong>నాశన అంశాలను</strong> <i>(\"R\"తో గుర్తు పెట్టబడింది)</i> శిథిలాల కింద యాదృచ్ఛికంగా <i>(ఇప్పటికే ఉన్న వస్తువులతో)</i>.<br/><i >4.</i> \"{{vagabond}}\" <strong>అక్షర కార్డ్</strong>ని మీ క్యారెక్టర్ కార్డ్ స్లాట్‌లో ఉంచండి. <InitialStartingItems>, </InitialStartingItems> మరియు <FinalStartingItem/> <strong>ప్రారంభ అంశాలను</strong> <i>(\"S\"తో గుర్తు పెట్టబడింది)</i>ని మీ సాట్చెల్‌లో మరియు వాటి మ్యాచింగ్ ట్రాక్‌లలో ఉంచండి, అవసరం మేరకు."
+    },
+    "warlord": {
+      "name": "నూరేళ్ల స్వామి(లార్డ్ అఫ్ హుండ్రేడ్స్)",
```

Comment:
I think the phonetic translation is a nice touch, however it makes the faction name too long to fit neatly in the interface. I would suggest just using the faction name from the official Leder games translation, or if that doesn't exist use just the literal translation without the phonetic one.